### PR TITLE
Set custom user agent on client

### PR DESCRIPTION
**Pull request by bfirsh**
See https://github.com/docker/compose/issues/3550

### CODE REVIEW

File: docker/constants.py [hunk 1]
```diff
@@ -12,3 +13,5 @@
     'is deprecated and non-functional. Please remove it.'
 
 IS_WINDOWS_PLATFORM = (sys.platform == 'win32')
+
+DEFAULT_USER_AGENT = "docker-py/%s" % version
```
Comment (shin-) [inline docker/constants.py]: Let's use `format` instead to be consistent with the rest of the codebase!

Comment (bfirsh) [inline docker/constants.py]: Fixed :)

**Comment by dnephin** [comment]
LGTM

**Comment by bfirsh** [comment]
Rebased. ping @shin- @aanand

**Comment by aanand** [comment]
LGTM

**Pull request merged**